src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/SNSActionPlanAppointmentServiceTest.kt

```diff
@@ -0,0 +1,74 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service
```

Comment:
This may seem slightly confusing. This file has just been renamed and therefore has not been 'added' as part of this pr. These are the tests for the old action plan appointments sns event service